includes/rest-api/class-wp-job-manager-controllers-status.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Declaration of our Status Model
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Controllers_Status
+ */
+class WP_Job_Manager_Controllers_Status extends WPJM_REST_Controller_Model
+	implements WPJM_REST_Interfaces_Controller {
+
+
+	/**
+	 * Setup
+	 */
+	function setup() {
+		$this->add_route( '/' )
+			->handler( 'index', 'index' );
+
+		$this->add_route( '/(?P<key>[a-zA-Z_]+)' )
```

Comment:
I like that you're doing a more explicit REGEX here. I have a tendency to forget to add the key names into my regex statements and this is far more readable!